libnavui-androidauto/build.gradle

```diff
@@ -40,8 +40,9 @@ dependencies {
 
     // This defines the minimum version of Navigation which is included in this SDK. To upgrade the
     // Navigation versions, you can specify a newer version in your downstream build.gradle.
-    releaseApi("com.mapbox.navigation:android:2.9.0-alpha.4")
-    debugApi project(":libnavigation-android")
+    // TODO uncomment when the release is ready
+    // api("com.mapbox.navigation:android:2.9.0-beta.1")
+    api project(":libnavigation-android")
```

Comment:
`MapboxAudioGuidance` has been introduced in `2.9.0` so the issue we're addressing here is specific to `2.9.0` and will not be an issue beyond